includes/wikia/Wikia.php

```diff
@@ -1264,9 +1262,9 @@ static public function verifyUserSecretKey( $url, $hash_algorithm = 'sha256' ) {
 		global $wgWikiaAuthTokenKeys;
 		wfProfileIn( __METHOD__ );
 
-		@list( $user, $signature1, $signature2, $public_key ) = explode("|", base64_decode( strtr($url, '-_,', '+/=') ));
+		@list( $user, $signature2, $public_key ) = explode("|", base64_decode( strtr($url, '-_,', '+/=') ));
```

Comment:
And either `$user` to `$username` or the other way so that the names match in verification and building methods?